src/qml/Statusbar.qml

```diff
@@ -38,7 +38,7 @@ import MeeGo.Connman 0.2
 Item {
     id: root
     z: 201
-    height: 48
+    height: root.width/12
```

Comment:
remove "root" here, I think it's a bit confusing, there is no need if it's a property of the same object